gnomad_qc/v5/data_ingestion/field_requirements.md

```diff
@@ -0,0 +1,231 @@
+| <span style="color:#999;">histograms.qual_hists.dp_hist_all.</span>n_smaller | `int64`         | Count of DP values falling below the lowest histogram bin edge, calculated on high quality genotypes. | —      | optional | |
+| <span style="color:#999;">histograms.qual_hists.dp_hist_all.</span>n_larger  | `int64`         | Count of DP values falling above the highest histogram bin edge, calculated on high quality genotypes.  | —      | optional | |
+| <span style="color:#999;">histograms.qual_hists.</span>gq_hist_alt                                                                                  | `struct { ... }`                                                        | Histogram for GQ in heterozygous individuals calculated on high quality genotypes.                                                                   | —                                              | optional      | will re-annotate                                                                                                                                                   |
+| <span style="color:#999;">histograms.qual_hists.gq_hist_alt.</span>bin_edges | `array<float64>` | Bin edges for the histogram of GQ in heterozygous individuals calculated on high quality genotypes are: `0 | 5 | 10 | 15 | 20 | 25 | 30 | 35 | 40 | 45 | 50 | 55 \| 60 | 65 | 70 | 75 | 80 | 85 | 90 | 95 | 100`                  | —      | optional | |
+| <span style="color:#999;">histograms.qual_hists.gq_hist_alt.</span>bin_freq  | `array<int64>`  | Bin frequencies for the histogram of GQ in heterozygous individuals calculated on high quality genotypes. The number of records found in each bin. | —      | optional | |
+| <span style="color:#999;">histograms.qual_hists.gq_hist_alt.</span>n_smaller | `int64`         | Count of GQ values in heterozygous individuals falling below the lowest histogram bin edge, calculated on high quality genotypes. | —      | optional | |
+| <span style="color:#999;">histograms.qual_hists.gq_hist_alt.</span>n_larger  | `int64`         | Count of GQ values in heterozygous individuals falling above the highest histogram bin edge, calculated on high quality genotypes. | —      | optional | |
+| <span style="color:#999;">histograms.qual_hists.</span>dp_hist_alt                                                                                  | `struct { ... }`                                                        | Histogram for DP in heterozygous individuals calculated on high quality genotypes.                                                                   | —                                              | optional      | will re-annotate                                                                                                                                                   |
```

Comment:
it just occurred to me that we always describe/document this as for het individuals, but it's for all variant carriers (het or hom), right? (and same for the GQ hist)

Reply:
oh good point, yeah it should be both but all our docs say het

https://github.com/broadinstitute/gnomad_methods/blob/a57e9e7c784f3e70f0d985fc19270c9898116b5b/gnomad/utils/annotations.py#L431